ui/lobby/src/variant.ts

```diff
@@ -42,6 +42,8 @@ const variantConfirms = {
     'This is a Backgammon game!\n\nThe aim of the game is to get all your pieces into your home, and then take them all off the board first.',
   nackgammon:
     'This is a Nackgammon game!\n\nThe aim of the game is to get all your pieces into your home, and then take them all off the board first.',
+  breakthroughtroyka: 'This is a Breakthrough game!\n\nThe first player who reach opposite rank wins',
```

Comment:
Probably need to use this description in one of the files above

Reply:
(And add a final "." here, to stay consistent)

Reply:
would change the second part to `The first player who reaches their last rank (opponent home row) wins.`